src/styleRe.re

```diff
@@ -15,9 +14,8 @@ type style =
       string [ | `value AnimatedRe.Value.t | `interpolation AnimatedRe.Interpolation.t];
 
 let combine a b => {
```

Comment:
No, it doesn't. I had mixed. 

Usually I either put something in separate styles and combine them in render, or put in one object in a `StyleSheet`.I have never combined styles in stylesheets because I always define styles inline. 

On the other hand it's a completely logical thing to do - I should add it. Also, it makes me come back to the list-style situation. I'd really like to define styles as just lists. Then our users wouldn't have to know all those functions.

What if we changed `StyleSheet` to the following:
```
let create (x: list ('a, Style.t)): StyleSheet.t 'a;
let get 'a : Style.t;
```